cloudinit/distros/freebsd.py

```diff
@@ -28,6 +34,13 @@ class Distro(cloudinit.distros.bsd.BSD):
     pkg_cmd_update_prefix = ["pkg", "update"]
     pkg_cmd_upgrade_prefix = ["pkg", "upgrade"]
     prefer_fqdn = True  # See rc.conf(5) in FreeBSD
+    home_dir = '/usr/home'
+
+    def _select_hostname(self, hostname, fqdn):
```

Comment:
With the `prefer_fqdn` class attribute, we no longer need to override this method.